scripts/fix-clippy.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+cd ../
```

Comment:
I usually call `./scripts/fix-clippy.sh` so that would break. You can [use this](https://stackoverflow.com/a/246128) then it should work from any path.

Reply:
Done